packages/fuels-test-helpers/src/node.rs

```diff
@@ -27,12 +27,14 @@ use tokio::process::Command;
 #[derive(Clone, Copy, Debug)]
 pub struct Config {
     pub addr: SocketAddr,
+    pub manual_blocks_enabled: bool,
 }
 
 impl Config {
     pub fn local_node() -> Self {
         Self {
             addr: SocketAddr::new(Ipv4Addr::new(127, 0, 0, 1).into(), 0),
+            manual_blocks_enabled: false,
```

Comment:
Would it be better to just have this set to true for our usual test setup methods?

Reply:
I'm not sure, since we are mostly not using this feature,it's pretty "potent/dangerous", and the error message is pretty clear.